src/encoded/static/components/search.js

```diff
@@ -1274,7 +1263,9 @@ export const SearchControls = ({ context, visualizeDisabledTitle, showResultsTog
             <div className="results-table-control__main">
                 <ViewControls results={context} additionalFilters={additionalFilters} />
                 {resultsToggle}
-                {showDownloadButton ? <BatchDownloadControls results={context} additionalFilters={additionalFilters} modalText={modalText} canDownload={canDownload} /> : ''}
+                {showDownloadButton
+                    ? <BatchDownloadControls results={context} additionalFilters={additionalFilters} modalText={modalText} canDownload={canDownload} />
```

Comment:
I just do this because the AirBnB examples do it this way. I don’t feel strongly about it, and without guidance I never really decided how I like to do it.